code/style.css

```diff
@@ -1,4 +1,297 @@
-html {
-  background: #0025ff;
-  color: #fff;
+* {
+  box-sizing: border-box;
+  }
+
+.body {
+  background:linear-gradient(#cbf7f7, #2d73d4, #020f3f, #020f3f,black, black, black);
+  color:rgb(226, 230, 231);
+  font-family: 'Times New Roman', Times, serif;
+  font-size: 16px;
+  letter-spacing: 0.07rem;
+  min-height: 3000px;
```

Comment:
If it weren't for your theme I would advice you not to set a specific height for the `body` but I'm choosing to interpret it as the real deep dive with that excessive space in the bottom 🤿